zha/application/discovery.py

```diff
@@ -314,7 +314,7 @@ def discover_quirks_v2_entities(self, device: Device) -> None:
                 endpoint.async_new_entity(
                     platform=platform,
                     entity_class=entity_class,
-                    unique_id=endpoint.unique_id,
+                    unique_id=f"{endpoint.unique_id}-{cluster.cluster_id}",
```

Comment:
I think this will break all existing v2 entities and it will require a migration of the HA entity registry.

Reply:
Is that migration a post-install operation?